tests/phpunit/unit/api/utilities/beansStrEndsWith.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Tests for beans_str_ends_with()
+ *
+ * @package Beans\Framework\Tests\Unit\API\Utilities
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\Utilities;
+
+use Beans\Framework\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_BeansStrEndsWith
+ *
+ * @package Beans\Framework\Tests\Unit\API\Utilities
+ * @group   api
+ * @group   api-utilities
+ */
+class Tests_BeansStrEndsWith extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
+	}
+
+	/**
+	 * Test beans_str_ends_with() should return false when case is wrong.
```

Comment:
should this be
`when case does not match`?
(Similar for other tests here and in str_starts_width)